metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/MathFunctionsTest.java

```diff
@@ -44,4 +44,16 @@ public void testAbs() {
     Assert.assertEquals((Double)run("ABS(value)", ImmutableMap.of("value", -10.5)), 10.5, 1e-7);
   }
 
+  @Test
+  public void testPower() {
+    Assert.assertEquals((Double)run("POWER(e,b)", ImmutableMap.of("e", 2.0, "b", 2.0)), 4.0, 1e-7 );
```

Comment:
Please put expected value first, test result value second, in assertEquals() calls.  Otherwise reported failures look funny.